### ISSUE TITLE
With contracts-upgradeable v5.0.0-rc.0 adminAddress is zero

### ISSUE THREAD

**Issue by SKYBITDev3** [OpenZeppelin/openzeppelin-upgrades#882]
When trying contracts-upgradeable v5.0.0-rc.0 and hardhat-upgrades, calling `forceImport` now fails because `adminAddress` is `0x0000000000000000000000000000000000000000`.

**Comment by frangio** [comment]
The Upgrades Plugins haven't yet been updated for 5.0 proxies.

**Comment by SKYBITDev3** [comment]
> The Upgrades Plugins haven't yet been updated for 5.0 proxies.

When will it be updated? I'm eager to try it.

**Comment by frangio** [comment]
This is likely to happen a few weeks after we release 5.0.

**Comment by ericglau** [comment]
@SKYBITDev3 Can you provide more information about what you mean by `adminAddress` is `0x0000000000000000000000000000000000000000`?  Where are you seeing that value being zero and how are you retrieving that value?

In https://github.com/OpenZeppelin/openzeppelin-upgrades/pull/883 we are enabling the import and upgrading of 5.0 proxies, but we did not encounter any issues for the actual import.  If possible, can you share what your scripts look like for deploying and importing the proxies?

**Comment by SKYBITDev3** [comment]
Here is my deployment script and failure location: https://github.com/SKYBITDev3/SKYBIT-Keyless-Deployment/blob/0e8051ba6bffb2a296ed2234f0429b8a5a3c1ba0/scripts/deployViaCREATE3-TESTERC20UG.js#L92

```
InvalidDeployment [Error]: No contract at address 0x0000000000000000000000000000000000000000 (Removed from manifest)
    at waitAndValidateDeployment (C:\Users\-\Documents\Projects\project\node_modules\@openzeppelin\upgrades-core\src\deployment.ts:251:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at fetchOrDeployGeneric (C:\Users\-\Documents\Projects\project\node_modules\@openzeppelin\upgrades-core\src\impl-store.ts:88:5)
    at fetchOrDeployAdmin (C:\Users\-\Documents\Projects\project\node_modules\@openzeppelin\upgrades-core\src\impl-store.ts:210:11)
    at simulateDeployAdmin (C:\Users\-\Documents\Projects\project\node_modules\@openzeppelin\hardhat-upgrades\src\utils\simulate-deploy.ts:19:32)
    at addAdminToManifest (C:\Users\-\Documents\Projects\project\node_modules\@openzeppelin\hardhat-upgrades\src\force-import.ts:111:3)
    at importProxyToManifest (C:\Users\-\Documents\Projects\project\node_modules\@openzeppelin\hardhat-upgrades\src\force-import.ts:89:5)
    at Proxy.forceImport (C:\Users\-\Documents\Projects\project\node_modules\@openzeppelin\hardhat-upgrades\src\force-import.ts:46:7)
    at main (C:\Users\-\Documents\Projects\project\scripts\deployViaCREATE3-TESTERC20UG.js:92:26)
```

I found `adminAddress` was 0 when I edited node_modules\@openzeppelin\hardhat-upgrades\src\force-import.ts to prnt out variable values.

**Comment by ericglau** [comment]
The issue is that `forceImport` is incorrectly inferring a UUPS proxy to be a transparent proxy.  This will be fixed by https://github.com/OpenZeppelin/openzeppelin-upgrades/pull/883, which we will release soon.

**Comment by ericglau** [comment]
@SKYBITDev3 This is fixed in [@openzeppelin/hardhat-upgrades@2.3.0](https://github.com/OpenZeppelin/openzeppelin-upgrades/releases/tag/%40openzeppelin%2Fhardhat-upgrades%402.3.0), you can update to that release to get the `forceImport` to work.

I've opened an additional PR #887 to give a more helpful error message in this kind of scenario.

**Comment by SKYBITDev3** [comment]
Thank you @ericglau. I've tried it and it's now working.
